plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/testing/TypeInfoHelper.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hudi.testing;
+
+import io.trino.metastore.HiveType;
```

Comment:
Is `HiveType` still the standard type class or we can depend on `TypeInfo` instead of `HiveType`?

Reply:
Got it.  Let's keep it.